src/cdk/platform/platform.ts

```diff
@@ -72,8 +72,11 @@ export class Platform {
   // Safari browsers will include the Safari keyword in their userAgent. Some browsers may fake
   // this and just place the Safari keyword in the userAgent. To be more safe about Safari every
   // Safari browser should also use Webkit as its layout engine.
+  // The userAgent for Chrome for iOS is the same as the Mobile Safari userAgent with
+  // CriOS/<ChromeRevision> instead of Version/<VersionNum>.
   /** Whether the current browser is Safari. */
-  SAFARI: boolean = this.isBrowser && /safari/i.test(navigator.userAgent) && this.WEBKIT;
+  SAFARI: boolean = this.isBrowser && /safari/i.test(navigator.userAgent) &&
+      !/crios/i.test(navigator.userAgent) && this.WEBKIT;
```

Comment:
Wouldn't this filter out plain Safari?

Reply:
I tested this change out in various Browserstack combinations. It seems to work as expected.

Reply:
I misread the expression- it now filters out Chrome on iOS (as advertised), which I'm not sure we actually want. Chrome on iOS really _is_ Safari in terms of rendering, so anything we do there we'd also want to do for iOS Chrome

Reply:
@jelbourn I thought about this too, but the JSDoc comment for the `SAFARI` class member convinced me that we only talk about the "browser" and not the rendering engine/foundation.

I agree that anything done for `iOS Safari` should be done for `iOS Chrome` too. Though this detection could be still achieved with all other detections provided by `Platform` right?

i.e. `platform.WEBKIT && platform.IOS`. Based on the Apple App Store guidelines, browsing the web always happens through Webkit, so the detection should be reliable? (even if someone uses another rendering engine through a non app store verified app).